python/route_guide/README.md

```diff
@@ -0,0 +1,281 @@
+// (degrees multiplied by 10**7 and rounded to the nearest integer).
+// Latitudes should be in the range +/- 90 degrees and longitude should be in
+// the range +/- 180 degrees (inclusive).
+message Point {
+  int32 latitude = 1;
+  int32 longitude = 2;
+}
+```
+
+## Generating client and server code
+
+Next you need to generate the gRPC client and server interfaces from your .proto service definition. You may do this using the protocol buffer compiler `protoc` with a special gRPC Python plugin. Make sure you've installed protoc and followed the gRPC Python plugin [installation instructions](https://github.com/grpc/grpc/blob/master/INSTALL) first):
+
+With `protoc` and the gRPC Python plugin installed use the following command to generate the Python code:
+
+```shell
+$ protoc -I . --python_out=. --grpc_out=. --plugin=protoc-gen-grpc=`which grpc_python_plugin` route_guide.proto
+```
+
+Running this command regenerates the route_guide_pb2.py file containing:
+- classes for the messages defined in route_guide.proto
+- abstract classes for the service defined in route_guide.proto
+   - `EarlyAdopterRouteGuideServicer` which defines the interface for implementations of the RouteGuide service
+   - `EarlyAdopterRouteGuideServer` which may be started and stopped
+   - `EarlyAdopterRouteGuideStub` which may be used to invoke RouteGuide RPCs
+- functions for application use
+   - `early_adopter_create_RouteGuide_server` which creates a server given a servicer object
+   - `early_adopter_create_RouteGuide_stub` which creates a stub object
+
+<a name="server"></a>
+## Creating the server
+
+First let's look at how you create a `RouteGuide` server. If you're only interested in creating gRPC clients, you can skip this section and go straight to [Creating the client](#client) (though you might find it interesting anyway!).
+
+Running a `RouteGuide` server breaks down into two work items:
+- Implementing the servicer interface generated from our service definition with functions that perform the actual "work" of the service.
+- Running a gRPC server to listen for requests from clients and transmit responses.
+
+You can find the example `RouteGuide` server in [grpc-common/python/route_guide/route_guide_server.py](https://github.com/grpc/grpc-common/blob/master/python/route_guide/route_guide_server.py).
+
+### Implementing RouteGuide
+
+`route_guide_server.py` has an `RouteGuideServicer` class that implements the generated interface `route_guide_pb2.EarlyAdopterRouteGuideServicer`:
+
+```python
+# RouteGuideServicer provides an implementation of the methods of the RouteGuide service.
+class RouteGuideServicer(route_guide_pb2.EarlyAdopterRouteGuideServicer):
+```
+
+`RouteGuideServicer` implements all the service methods. Look at the simplest type first, `GetFeature`, which just gets a `Point` from the client and returns the corresponding feature information from its database in a `Feature`.
```

Comment:
all the service methods -> all the `RouteGuide` service methods
Look -> Let's look